samples/server/petstore/cpp-pistache/api/UserApi.cpp

```diff
@@ -78,11 +75,11 @@ void UserApi::create_user_handler(const Pistache::Rest::Request &request, Pistac
 void UserApi::create_users_with_array_input_handler(const Pistache::Rest::Request &request, Pistache::Http::ResponseWriter response) {
 
     // Getting the body param
-    std::vector<User> user;
+    std::vector user;
```

Comment:
This is strange, because this new change here will not compile.

Reply:
I've regenerated the samples and this issue seems to be gone.